addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -230,7 +230,7 @@ const FieldEditor = FormEditor.extend({
             labelPosition: this._getLabelPosition(),
             labelWidth: this.$target[0].querySelector('.s_website_form_label').style.width,
             multiPosition: multipleInput && multipleInput.dataset.display || 'horizontal',
-            col: [...this.$target[0].classList].filter(el => el.match(/^col-/g)).join(' '),
+            col: [...this.$target[0].classList].filter(el => el.match(/^(col|offset)-/g)).join(' '),
```

Comment:
I'm not entirely sure adding offset classes here is the right move.

The only place in the code where I could find this `format.col` is in the template `website.form_field` (as `field.formatInfo.col`), and it clearly expects only a `col-` class (because it defaults to `col-12` if there is none).
Adding the offsets here could potentially mean (although I don't know how) that there would be no `col-` class but an offset, and so `col-12` wouldn't be applied either, causing UI bugs.

The pros of doing it like that: one-line fix, the conflict doesn't seem so likely, very stable-friendly.

Another option would be to add another key `offset` on the format, and call `formatInfo.offset` in the template.
Cons: less stable-friendly.
But it's still only a 2-line diff, unlikely to create conflicts with existing DBs (I think?), and more sturdy.

Not sure on this one.

Reply:
That said, it might be worth adding a step in a test to check that kind of behavior. :)